### PR TITLE
refactor Apply SSC timeouts to all SSC requests

### PR DESCRIPTION
**Pull request by vrulevskyi**

### CODE REVIEW

File: src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java [hunk 1]
```diff
@@ -1542,6 +1548,42 @@ public boolean configure(StaplerRequest req, JSONObject o) throws FormException
 				breakdownPageSize = DEFAULT_PAGE_SIZE;
 			}
 
+			try {
+				String connectTimeoutString = o.getString("connectTimeout");
```
Comment (hiro0824) [inline src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java:1552]: Sorry, I know this is not named by you, but can you please rename o to something else like obj, json or jsonObject? 'o' is a little too short and hard to understand...

**Review by hiro0824 — APPROVED** [review]
Can you please rename the variable?
Other than that, looks good. and I like your code. It's pretty clean. 😄

File: src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java (hunk 1 above)
Comment (vrulevskyi) [reply src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java]: Renamed

File: src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java [hunk 2]
```diff
@@ -1516,22 +1522,22 @@ public boolean configure(StaplerRequest req, JSONObject o) throws FormException
 				}
 			}
 			try {
-				String tokenParam = o.getString("token").trim();
+				String tokenParam = jsonObject.getString("token").trim();
```
Comment (tps193) [inline src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java:1525]: Sorry, I am not really familiar with net.sf.json, but some libraries return null for getJSONObject() if object doesn't exist. Just want to make sure that we are safe here and String.trim() wouldn't throw Null Pointer exception for some cases

**Review by tps193 — APPROVED** [review]
Look ok

File: src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java (hunk 2 above)
Comment (vrulevskyi) [reply src/main/java/com/fortify/plugin/jenkins/FortifyPlugin.java:1525]: I found in the [documentation](http://json-lib.sourceforge.net/apidocs/net/sf/json/JSONObject.html#getString(java.lang.String)) that method `getString(String)` throws the `JSONException` if key is not found. This error is caught thereafter in the code for all the occurrences of usage of that method in `FortifyPlugin` class. Marking this comment as resolved.

**Comment by vrulevskyi** [comment]
Merged.

**Pull request merged**